README.md

```diff
@@ -115,8 +115,8 @@ tasks.dokkaHtml.configure {
 
 Groovy
 ```kotlin
-tasks.named("dokkaHtml") {
-    outputDirectory.set(buildDir.resolve("dokka"))
+tasks.dokkaHtml {
+    outputDirectory = "$buildDir/dokka"
```

Comment:
The `.set` is still necessary in Groovy also. We are trying to investigate this, soon.

Reply:
Somehow it doesn't work if I do it with `.set`

```
dokkaHtml {
    outputDirectory.set("$buildDir/dokka")
}
```
results in error `No signature of method: build_2u87vgqvma4krbutdh5r4jh3w.dokkaHtml() is applicable for argument types: (build_2u87vgqvma4krbutdh5r4jh3w$_run_closure8) values: [build_2u87vgqvma4krbutdh5r4jh3w$_run_closure8@3cbe711b]`

but
```
dokkaHtml {
    outputDirectory = "$buildDir/dokka"
}
```
works just fine.

Oh, and the `tasks.` in front of the `dokkaHtml` seems to be redundant

Reply:
I found the reason why it's not working

the `outputDirectory` property was type `Property<File>` in [`master`](https://github.com/Kotlin/dokka/blob/master/runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/AbstractDokkaTask.kt) branch
but it was changed to `String` in [`1.4.0-RC`](https://github.com/Kotlin/dokka/blob/1.4.0-rc/runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/AbstractDokkaTask.kt) branch

Reply:
I think you have to change the Kotlin DSL part too...?